x-pack/plugins/ml/public/explorer/explorer_utils.js

```diff
@@ -388,31 +388,41 @@ export async function loadAnnotationsTableData(selectedCells, selectedJobs, inte
     return Promise.resolve([]);
   }
 
-  const resp = await ml.annotations.getAnnotations({
-    jobIds,
-    earliestMs: timeRange.earliestMs,
-    latestMs: timeRange.latestMs,
-    maxAnnotations: ANNOTATIONS_TABLE_DEFAULT_QUERY_SIZE
-  });
+  return new Promise((resolve) => {
```

Comment:
as this no longer uses `await`, this function doesn't need to be `async`

Reply:
Thanks, fixed in d178eb3.